iceoryx_hoofs/include/iceoryx_hoofs/log/building_blocks/logformat.hpp

```diff
@@ -46,26 +50,82 @@ struct LogBuffer
 namespace internal
 {
 template <typename T>
-constexpr const char* logFormatDec();
+constexpr const char* logFormatDec() noexcept;
+template <>
+inline constexpr const char* logFormatDec<signed char>() noexcept;
```

Comment:
Please remove the `inline` keyword since it is meaningless in the function declaration.